terraform/modules/front_door/cloudfront.tf

```diff
@@ -4,10 +4,7 @@ locals {
 }
 
 #tfsec:ignore:aws-cloudfront-enable-logging:TODO we will be implementing logging later
-#tfsec:ignore:aws-cloudfront-enable-waf:TODO CLDC-2546
 resource "aws_cloudfront_distribution" "this" {
-  #checkov:skip=CKV2_AWS_47:TODO CLDC-2546 when setting up WAF it should be configured appropriately to mitigate against the Log4j vulnerability https://docs.bridgecrew.io/docs/ensure-aws-cloudfront-attached-wafv2-webacl-is-configured-with-amr-for-log4j-vulnerability
```

Comment:
This todo had a link to [here](https://docs.bridgecrew.io/docs/ensure-aws-cloudfront-attached-wafv2-webacl-is-configured-with-amr-for-log4j-vulnerability), which suggested adding the rules `AWSManagedRulesKnownBadInputsRuleSet` (which we already have) and `AWSManagedRulesAnonymousIpList` (added below) in our WAF acl to mitigate a log4j vulnerability

`AWSManagedRulesAnonymousIpList` can block requests from VPNs and Proxies (see info [here](https://docs.aws.amazon.com/waf/latest/developerguide/aws-managed-rule-groups-ip-rep.html)) which might cause issues though, so I'll check this works ok locally (since we use GlobalProtect) and we may need to keep an eye on it down the line

Reply:
This has been applied to staging and my local machine (with GlobalProtect VPN) seems to work OK

Reply:
I'm a bit concerned that end users at local authorities / housing associations that use VPNs may get caught by this too (i.e. it's probably not enough to just know that we can access it while on our VPN). I'm not sure how much you've looked into this - do you have a sense of how likely it is that this becomes a problem?

Reply:
Yeah I agree with the concern 😅. There's nothing concrete from AWS about how much this affects legitimate users of VPNs. The best info I was able to find was the below (not very good):

- A comment on a reddit post saying half their customers were blocked [here](https://www.reddit.com/r/aws/comments/smrxvz/how_to_remove_address_from/)
- A question on AWS re:post clarifying about VPNs [here] (https://repost.aws/questions/QUGXTX6M9RTBS3BQnvIiJFuA/aws-waf-specifically-block-tor)

I suspect you'll want to leave the rule out in this case, and add a code comment about not being able to use it?

Reply:
Drat - you'd really think AWS would be able to be a bit cleverer about it, maybe there isn't really a way to be.

Probably something to leave out in that case.

Reply:
I've removed the rule 😀👍